src/database/datatools/scheduletypelimits.py

```
@@ -0,0 +1,131 @@
+                                architecture_type: str,
+                                lower_limit_value: float,
+                                upper_limit_value: float,
+                                numeric_type: str,
+                                unit_type: str) -> None:
+    
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "schedule_type_limits"
+
+    sql = f"INSERT INTO {table_name} (name, latitude, longitude, architecture_type, lower_limit_value, upper_limit_value, numeric_type, unit_type, datetime) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)"
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)